sky/adaptors/azure.py

```diff
@@ -14,19 +17,25 @@
 
 _session_creation_lock = threading.RLock()
 
+# There is no programmatic way to get current account details anymore.
+# https://github.com/Azure/azure-sdk-for-python/issues/21561
+
+
+def _get_account():
+    return json.loads(
+        run('az account show -o json',
+            stdout=subprocess.PIPE,
+            stderr=subprocess.DEVNULL).stdout.decode())
```

Comment:
Should we expose the stderr here and add proper error handling? maybe raise an runtime error if json parse error and print out the error message